doc/whats_new/v0.22.rst

```diff
@@ -137,6 +137,15 @@ Changelog
 :mod:`sklearn.gaussian_process`
 ...............................
 
+- |Feature| :func:`gaussian_process.GaussianProcessClassifier.log_marginal_likelihood`
+  and :func:`gaussian_process.GaussianProcessRegressor.log_marginal_likelihood` now
+  accept a ``clone_kernel=True`` keyword argument. When set to ``False``,
+  the kernel attribute is modified, but may result in a performance improvement.
+  :func:`gaussian_process.GaussianProcessClassifier.fit` and
```

Comment:
I think you don't need lines 144-146

Reply:
OK! I'll remove 144-146. BTW, do you know the reason why `doc-min-dependencies` job is failed at CircleCI?

Reply:
don't worry about the circleCI, we know about it. Conda has removed some old packages, that's why it fails.

Reply:
I got it 👍  Thank you!